.github/workflows/run-clang-format.sh

```diff
@@ -15,8 +15,9 @@
 
 FORMAT_OUT=${TMPDIR:-/tmp}/clang-format-diff.out
 
```

Comment:
Potential future improvement -- I wonder if it is worth checking the clang-format version is correct?